Optional field rating in add command does not do validation

<img width="1572" height="771" alt="Image" src="https://github.com/user-attachments/assets/a088397c-5586-4356-b9b3-6d4377214f4c" />

Ignore the UI warnings, but when trying out the command `add n/dd p/88882222 e/aa@gmail.com a/Blk 123, Street, #11-111, S213123 r/11` will create an exception with no error feedback to GUI

Exception:
```
Caused by: java.lang.reflect.InvocationTargetException
Caused by: java.lang.IllegalArgumentException: Ratings should only contain numbers, and be an integer between 1 to 10
```

--- a/src/main/java/seedu/bitebuddy/logic/parser/ParserUtil.java
+++ b/src/main/java/seedu/bitebuddy/logic/parser/ParserUtil.java
@@ -152,11 +152,13 @@ public class ParserUtil {
 
         // Case where >= 1 Rate's CLI tag have been detected, take latest one
         Integer rating;
+        Rate rate;
         try {
             rating = Integer.valueOf(mostRecentRating);
-        } catch (NumberFormatException e) {
+            rate = new Rate(rating);
+        } catch (IllegalArgumentException e) {
             throw new ParseException(Rate.MESSAGE_CONSTRAINTS);
         }
-        return new Rate(rating);
+        return rate;
     }
 }

--- a/src/main/java/seedu/bitebuddy/model/foodplace/Rate.java
+++ b/src/main/java/seedu/bitebuddy/model/foodplace/Rate.java
@@ -76,7 +76,7 @@ public class Rate {
 
     @Override
     public String toString() {
-        return String.format("<%d/%d>", value, MAX);
+        return String.format("< %d/%d >", value, MAX);
     }
 
     @Override

--- a/src/main/java/seedu/bitebuddy/ui/FoodplaceCard.java
+++ b/src/main/java/seedu/bitebuddy/ui/FoodplaceCard.java
@@ -57,13 +57,13 @@ public class FoodplaceCard extends UiPart<Region> {
         address.setText(foodplace.getAddress().value);
         email.setText(foodplace.getEmail().value);
         note.setText(String.valueOf(
-                foodplace.getNote().value.length() != 0 ? foodplace.getNote().value : ">>No notes yet<<"
+                foodplace.getNote().value.length() != 0 ? foodplace.getNote().value : "-- No notes yet --"
         ));
         foodplace.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
         rate.setText(String.valueOf(
-                foodplace.getRate().isSet() ? foodplace.getRate().toString() : ">>No Rating yet<<"
+                foodplace.getRate().isSet() ? foodplace.getRate().toString() : ">> No Rating yet <<"
         ));
     }
 }

